pkg/arvo/app/graph-store.hoon

```diff
@@ -627,22 +627,27 @@
     |=  [=graph:store mark=(unit mark:store)]
     ^-  [? _state]
     ?~  mark   [%.y state]
-    =/  has-dais  (~(has by validators) u.mark)
-    =/  =dais:clay
-      ?:  has-dais
+    =/  has-validator  (~(has by validators) u.mark)
+    =/  validate=$-(indexed-post:store indexed-post:store)
+      ?:  has-validator
```

Comment:
We have to use `?:` instead of `fall` because `fall`s arguments will be evaluated eagerly?

Reply:
I'm doing a style pass separately (and all at once) as a part of this sprint. Not sure if they're eagerly evaluated! Will evaluate using that then

Reply:
If you need a lazily-evaluated default, use `+bond`.